resources/views/event/display_includes/event_details.blade.php

```diff
@@ -80,7 +80,17 @@
                 <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i> This event is not organized by S.A. Proto
             </li>
         @endif
-
+        @if (!Auth::check()&&$event->activity !== null)
```

Comment:
I do not get the difference between the two possible buttons. The text seems to say the same thing. But in case you really want to show different text you could use something like: `{{ isset($event->activity) ? <text> : <other_text> }}` so you don't have to define the same button twice.

Reply:
I have now made the division between the buttons more clear and divided the functionality. I want to have a different one reminding you to signup as neccesary. I hope my new commit clears it up 😀